go/protocol/keybase1/extras.go

```diff
@@ -742,6 +742,85 @@ func (s SigID) ToShortID() string {
 	return encode(s.ToBytes()[0:SIG_SHORT_ID_BYTES])
 }
 
```

Comment:
above this in the file, there's a method `SigIDFromString` which is called a bunch, e.g. in chain link parsing. it would be great to push all of that code onto the same flows you have for SigIDBase... but short of that (seems like that would be kind of a lot of work for dubious benefit), can we rename it so it's clear it only ever returns v1 suffixes? maybe `LegacySigIDFromString` or `V1SigIDFromString`? 

huh. and actually it looks like `SigIDFromBytes` and `SigIDFromSlice` aren't called by anything anymore.